manifests/init.pp

```diff
@@ -38,17 +38,17 @@
     default        => '\[\033[31m\]',
   }
 
-  if $bashpackage {
-    package { $bashpackage:
+  if $bashpackage and ! defined(Package['bash']) {
+    ensure_packages([$bashpackage], {
```

Comment:
With the `ensure_packages` here I am fairly certain the `! defined` becomes overkill...

Reply:
or is this to avoid two packages solving the same problem?

Reply:
This is for the alias in the case that `$bashpackage` isn't “bash.” (I just checked and `ensure_packages` doesn't take the alias into account.)

I figure if something is called “bash” then it's probably good enough.